platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminContext.java

```diff
@@ -289,8 +293,13 @@ private ImportMigrationConfigurationAdminEntry proxy(ImportMigrationEntry entry)
             .record(
                 new MigrationException("Import error: missing pid from configuration [%s].", path));
       } else {
-        final String bundleLocation =
-            Objects.toString(properties.remove(ConfigurationAdmin.SERVICE_BUNDLELOCATION), null);
+        String bundleLocation = null;
+        if (restoreWithBundleLocation) {
+          bundleLocation =
+              Objects.toString(properties.remove(ConfigurationAdmin.SERVICE_BUNDLELOCATION), null);
+        } else {
```

Comment:
❗️ The bundle location property should still be removed from the `properties` object even though you are ignoring it